iot/api/CloudIotTest/CloudIotTest.cs

```diff
@@ -28,7 +28,7 @@ public class CommonTests
         private static readonly string s_projectId = Environment.GetEnvironmentVariable("GOOGLE_PROJECT_ID");
         private static readonly string s_regionId = "us-central1";
         private static readonly string s_serviceAccount = "serviceAccount:cloud-iot@system.gserviceaccount.com";
-
+        private static readonly string s_testID = Guid.NewGuid().ToString().Split("-")[0];

```

Comment:
There is existing method (RandomName) to create randomness in this codebase:
https://github.com/GoogleCloudPlatform/dotnet-docs-samples/blob/master/testutil/TestUtil.cs

I'll leave it up to your judgement whether to leave it as is or utilize TestUtil.